tests/integration/test_agent.py

```diff
@@ -85,7 +84,7 @@ def test_ipython():
     # Execute the task
     task = "Use Jupyter IPython to write a text file containing 'hello world' to '/workspace/test.txt'. Do not ask me for confirmation at any point."
     final_agent_state = asyncio.run(main(task, exit_on_message=True))
-    assert final_agent_state == AgentState.FINISHED
+    print('final_agent_state:', final_agent_state)
```

Comment:
We probably shouldn't remove this assert, should we?

Reply:
This was me horsing around w/ the tests 😄 reverted